sequence_processing_pipeline/TRIntegrateJob.py

```diff
@@ -161,3 +158,27 @@ def _generate_job_script(self):
                 "output_dir": self.output_path}))
 
         return job_script_path
+
+    def audit(self):
```

Comment:
Can this and the other `audit` method be part of a higher method or maybe be in util?

Reply:
Not really, no. I did move the audit methods that were largely the same down into the base Job class though, based on your previous suggestion. Now every job has a basic audit() method that parses all of the files in the log directory for lines that include 'error', case insensitive. The consequence of that is that if you have a better method for parsing a particular job's output then you need to override the method as is done here.